terraform/deploy-terraform.sh

```diff
@@ -27,6 +27,8 @@ echo "releaseNumber: $releaseNumber";
 echo "releaseEnvironment: $releaseEnvironment";
 echo "runningInOctoDeploy: $runningInOctoDeploy";
 
+searchLambdaLocation="../lambdas/search-lambda.zip"
```

Comment:
This variable doesn't seem to be being used anywhere?

Reply:
yes - line 64